### PR TITLE
US134319: Move Tests

### PR DESCRIPTION
**Pull request by Scy-D2L**
Moves component tests from the 'perceptual' folder to allow unit/vdiff tests to exist side-by-side.

Fixes LCB vdiff tests not including submission dropdown.

Adds a (currently skipped) axe test for the LCB. This is a PoC for the way component-specific axe tests should be setup, but we need to figure out how to load HM data into a component before we can run axe/unit tests on the components in general.

Simplifies running local vdiff tests with the commands:
`npm run vdiff`
`npm run vdiff:goldens`

### CODE REVIEW

**Review by JamieTran — APPROVED** [review]
lgtm, the new vdiff command looks super useful!

File: test/components/header/consistent-evaluation-lcb.axe.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+const USER_HREF = './data/user.json';
+const ENROLLED_USER_HREF = './data/user-enrolled.json';
+const SUBMISSION_LIST = [
+	{ 'href': './data/submission3.json' },
+	{ 'href': './data/submission2.json' },
+	{ 'href': './data/submission1.json' }];
+
+//Skipped until we find out how to load test data
+describe.skip('learner-context-bar-axe', () => {
```
Comment (jeroach) [inline test/components/header/consistent-evaluation-lcb.axe.js:12]: So these are net new tests, but we want them skipped for now? Just confirming :)

Comment (jeroach) [reply test/components/header/consistent-evaluation-lcb.axe.js:12]: I should read the description lol, sorry

**Review by jeroach — APPROVED** [review]
LGTM - sorry for the delay - with the github errors I had stopped checking until I saw the email notification from JT's review

**Pull request merged**